Makefile

```diff
@@ -273,31 +273,31 @@ $(CNI_BIN_ROOT):
 
 BRIDGE_BIN?=$(BINPATH)/bridge
 $(BRIDGE_BIN):
-	GOBIN=$(dir $@) GO111MODULE=off go get -u github.com/containernetworking/plugins/plugins/main/bridge
+	GOBIN=$(dir $@) go install github.com/containernetworking/plugins/plugins/main/bridge@v1.1.0
```

Comment:
Why do we need to pin the version here. Can we not use `@latest`? This will need separate update mechanism otherwise

Reply:
I had to pin the plugins to some older versions to pass integ-tests, which still have the CNI configured for older versions

Reply:
Besides CNI, there are some other packages such as `git-validation` etc that have their versions pinned.

Reply:
Yeah. For toolchain binaries, it's better to pin to a known good version otherwise upgrade may (surprisingly) break our build.